packages/data/README.md

```diff
@@ -973,6 +973,25 @@ _Returns_
 
 <!-- END TOKEN(Autogenerated API docs) -->
 
+### batch
+
+As a response of `dispatch` calls, WordPress data based applications updates the connected components (Components using `useSelect` or `withSelect`). This update happens in two steps:
+
+-   The selectors are called with the update state.
+-   If the selectors return values that are different than the previous (strict equality), the component rerenders.
+
+As the application grows, this can become costful, so it's important to ensure that we avoid running both these if possible. One of these situations happen when an interaction requires multiple consisecutive `dispatch` calls in order to update the state properly. To avoid rerendering the components each time we call `dispatch`, we can wrap the sequential dispatch calls in `batch` which will ensure that the components only call selectors and rerender once at the end of the sequence.
+
+_Usage_
+
+```js
```

Comment:
Based on the usage, I reason that the usage example should be closer to:
```js
import { useRegistry } from '@wordpress/data';

function Component() {
	const registry = useRegistry();

	function callback() {
		// This will only rerender the components once.
		registry.batch( () => {
 			registry.dispatch( someStore ).someAction();
 			registry.dispatch( someStore ).someOtherAction();
 		} );
	}
	return <button onClick={ callback }>Click me</button>;
}
```

Reply:
Does it make any difference when the are actions dispatched from multiple stores?

Reply:
That is indeed a better example :)